src/frontend/utils/validators.ts

```diff
@@ -64,7 +64,10 @@ const finnesDatoFørFødselsdato = (person: IGrunnlagPerson, fom: string, tom?:
     const fomDato = familieDayjs(new Date(fom));
     const tomDato = tom ? familieDayjs(new Date(tom)) : undefined;
 
-    return fomDato.isBefore(fødselsdato) || (tomDato ? tomDato.isBefore(fødselsdato) : false);
+    return (
+        fomDato.isBefore(fødselsdato, 'day') ||
```

Comment:
Hva er forskjellen mellom day og date?

date | D | 00:00 today
-- | -- | --
day | d | 00:00 today

Reply:
vet ikke, men hvis du sender lenka til hvor du har dette fra kan jeg se

Reply:
https://day.js.org/docs/en/manipulate/start-of#list-of-all-available-units

Reply:
mulig det er day of week, date høres mer riktig ut. Bra catch!

Reply:
Kan vurdere å sette enhet også på formatter.ts linje 77.